src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/SamlTokenValidationParameters.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using CoreWCF.IdentityModel.Selectors;
+using Microsoft.IdentityModel.Tokens;
+using MsIdentityTokens = Microsoft.IdentityModel.Tokens;
+
+namespace CoreWCF.IdentityModel.Tokens
+{
+    internal class SamlTokenValidationParameters
+    {
+        private const string CERTIFICATE_VALIDATOR = "Certificatevalidator";
+        private const string SAML_SECURITY_TOKEN = "SamlSecurityToken";
```

Comment:
I presume the V should be upper case? Also these are used with a property bag which other code might put things in so you need to worry about name collisions. Prefix these key names with the namespace and class name to avoid potential collisions in the property bag.